meinberlin/apps/users/fields.py

```diff
@@ -37,7 +37,7 @@ class EmailFileField(forms.FileField):
     widget = widgets.FileInput
     # Find possible email strings. Emails may be quoted and separated by
     # whitespaces, commas or semicolons.
-    email_regex = re.compile(r'[^\s;,"\']+@[^\s;,"\']+\.[a-z]{2,}')
+    email_regex = re.compile(r'[^\s;,"\'<]+@[^\s;,"\']+\.[a-z]{2,}')
```

Comment:
the regex should match the `>` after the add, too.
although i cant find a usefeul example where this would be a problem o.0